beta/adobe-journey-optimizer/push-messaging-guide.md

```diff
@@ -0,0 +1,81 @@
+
+Static data that identifies the user of the mobile app such as a push token, is stored against the user's profile as record data while the interactions the user does with push notifications are tracked as time-series events data. 
+
+#### Adobe Journey Optimizer
+
+Once your mobile app integrations with above mentioned components are in place and your customer profiles are available as Real-time customer profiles, you can take advantage of powerful audience segmentation capabilities in [Adobe Journey Optimizer](https://experienceleague.adobe.com/docs/journey-optimizer/using/ajo-home.html) to ensure the optimal experience for each person.
+
+
+
+### Push Data Flow Steps
+
+Refer to the data flow diagram to identify what each of these bullets represent :
+
+1. The customer develops a mobile app on Android or iOS that they would release to their users. In order to use the push capability provided by push providers i.e APNS by Apple and FCM by Google, the mobile app registers itself and enables the push capability.
+2. The push providers generate and send a push token to the mobile app. A Push Token is an identifier that senders use to target specific devices with a push notification.
+3. The mobile app is integrated with Adobe Mobile SDK which exposes various extensions and APIs. The Messaging extension exposes an API to register the push token into Adobe Experience Platform against customer's profile.
+   Once the mobile app is ready, it is configured in Adobe Launch > App Configurations along with the credentials.
+4. The marketer now authors a push notification in Adobe Journey Optimizer > Messages against the mobile app registered. 
+5. The marketer orchestrates a customer journey defining the flow of events and actions. To send a push notification at a stage of the journey, the marketer adds an action of type 'Message' and associates it with the message authored in the previous step. 
+6. Whenever a customer profile qualifies to receive the push notification for any reason say on trigger of an event or on qualification of a segment, the message is personalized against the profile, if applicable.
+7. The personalized push message is sent for further processing to the Push delivery service.
+8. The push delivery service validates the credentials of the app associated with the message.
+9. The message is sent to the push providers for delivery to the mobile app against the specific push token and credentials.
+10. The push providers send a feedback suggesting whether the message was successfully delivered to the provider or not. In case not, the relevant error message is part of the feedback. This feedback is sent to Adobe reporting for the customer to view in their Journey Live and Global Reports.
```

Comment:
"If the message was not successfully delivered, the relevant error message is sent as part of the feedback."